### ISSUE TITLE
Implementing the strict zero-shot setting on PromptAlign.

### ISSUE THREAD

**Issue by Bostoncake** [Razaimam45/TTL-Test-Time-Low-Rank-Adaptation#1]
Hi! I have read your paper and found that you implemented a strict zero-shot setting to evaluate TTL and its competing methods. I'd like to know how to implement PromptAlign in this setting since PromptAlign demands the distribution information of tokens on source data to perform the alignment. Without source data and pre-trained weights, how to get access to the distribution? 

Thank you.

**Comment by hananshafi** [comment]
Hello, We dont make any changes to the Alignment loss in PromptAlign. We use the same Mean and standard deviations of the source data from PromptAlign as they have already open-sourced that, for alignment loss. However, we make changes with the way prompts are initialized in the PromptAlign. They initialize the prompts from few-shot Imagenet pretrained weights, which invalidates the strict zero-shot criteria. Instead, we initialize the prompts of PromptAlign with random weights for fair evaluation. 

I hope this clears your doubts.
